{
 "PRed": [
  {
   "PR": {
    "__lazy_json__": "pr_json/238981701.json"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "CompilerRebuild",
    "migrator_version": 1,
    "name": "Python 3.7, GCC 7, R 3.5.1, openBLAS 0.3.2"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "__lazy_json__": "pr_json/254233424.json"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "2.16.11.2"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "__lazy_json__": "pr_json/299554357.json"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "RBaseRebuild",
    "migrator_version": 0,
    "name": "r-base-3.6.1"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "__lazy_json__": "pr_json/339353470.json"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "2.16.17"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "__lazy_json__": "pr_json/413615667.json"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 2,
    "migrator_version": 0,
    "name": "r400"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  }
 ],
 "bad": false,
 "pinning_version": "2020.05.04.19.59.17",
 "pre_pr_migrator_attempts": {
  "r410": 1
 },
 "pre_pr_migrator_status": {
  "r410": "not solvable (<a href=\"https://github.com/regro/autotick-bot/actions/runs/1005136621\">bot CI job</a>): master: ['linux_64_r_base4.1: Encountered problems while solving:\\n  - package r-refgenome-1.7.3-r351h29659fb_1000 requires r-base >=3.5.1,<3.5.2.0a0, but none of the providers can be installed\\n', 'osx_64_r_base4.1: Encountered problems while solving:\\n  - package r-refgenome-1.7.7-r35hf99fc2c_1 requires r-base >=3.5,<3.6.0a0, but none of the providers can be installed\\n', 'win_64_r_base4.1: Encountered problems while solving:\\n  - package r-refgenome-1.7.3-r351h6115d3f_0 requires r-base >=3.5.1,<3.5.2.0a0, but none of the providers can be installed\\n']"
 },
 "smithy_version": "No azure token. Create a token and\nput it in ~/.conda-smithy/azure.token\n3.7.0\n"
}